CHANGES.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added `load_best` attribute to `Checkpoint` callback to automatically load state of the best result at the end of training
+- Added `load_best` attribute to `EarlyStopping` callback to automatically load module weights of the best result at the end of training
```

Comment:
We had a skorch release recently, resulting in the addition of a new section in the CHANGES.md. Could you please merge with/rebase against master. Your change will be at the very top of the new release :)

Reply:
Sure!